### PR TITLE
books: fixed arXiv eprints and collaboration rules

### PR DESCRIPTION
**Pull request by kprzerwa**
* add ignored keys
* tweak record match queries

### CODE REVIEW

File: cds_dojson/marc21/fields/books/base.py [hunk 1]
```diff
@@ -212,28 +212,28 @@ def corporate_authors(self, key, value):
...
                                        'type': 'ORGANISATION'})
-    self['authors'] = _corporate_authors
-    raise IgnoreKey('corporate_authors')
+    return _corporate_authors
 
 
-# TODO waiting for verification
-@model.over('collaborations', '^710__')
-@replace_in_result('Collaboration', '', key='value')
+@model.over('authors', '^710__')
```
Comment (ntarocco) [inline cds_dojson/marc21/fields/books/base.py:223]: isn't authors stored in `700`?

File: cds_dojson/marc21/models/books/multipart.py [hunk 2]
```diff
@@ -20,21 +20,22 @@
...
 from .base import model as books_base
-from .base import CDSOverdoBookBase, COMMON_IGNORE_FIELDS
 
 
 class CDSMultipart(CDSOverdoBookBase):
     """Translation Index for CDS Books."""
 
     __query__ = '(690C_:BOOK OR 690C_:"YELLOW REPORT" OR ' \
-                '690C_:BOOKSUGGESTION OR 980__:PROCEEDINGS OR 980__:PERI OR ' \
+                '980__:PROCEEDINGS OR ' \
```
Comment (ntarocco) [inline cds_dojson/marc21/models/books/multipart.py:31]: for the `BOOKSUGGESTION`, we will delete them from CDS???

Comment (kprzerwa) [reply cds_dojson/marc21/models/books/multipart.py:31]: in one of the discussion I as told by Anne that we don't need to migrate this collection

File: cds_dojson/marc21/fields/books/base.py (hunk 1 above)
Comment (kprzerwa) [reply cds_dojson/marc21/fields/books/base.py:223]: they are in 710, 700 and 100 :)

**Pull request merged**